README.md

```diff
@@ -234,7 +234,7 @@ Response body
 #### Get access token for an existing entity
 Request
 ```shell
-curl --request POST 'http://<CAPIF_HOSTNAME>/gettoken' --header 'Content-Type: application/json' --data '{
+curl --request POST 'http://<CAPIF_HOSTNAME>:8080/gettoken' --header 'Content-Type: application/json' --data '{
```

Comment:
not always will be port 8080, for example, at deployment on openshift the port is 80, this is why in previous version of curl command I didn't setup a port at request to http endpoint. Maybe we can put something of <CAPIF_HTTP_PORT>=:8080